src/containers/layout/navbar/NavBar.jsx

```diff
@@ -60,10 +59,10 @@ const Navbar = () => {
   return (
     <Box sx={styles.header}>
       <Button
-        component={Link}
+        component={HashLink}
         size='small'
         sx={styles.logoButton}
-        to={guestRoutes.home.path}
+        to={guestRoutes.home.path + '#welcome'}
```

Comment:
hash should be added to routes, not hardcoded in component

Reply:
added it to route constants instead